app/views/api/v2/layouts/index_layout.json.erb

```diff
@@ -4,6 +4,9 @@
   "page": <%= metadata_page.to_json %>,
   "per_page": <%= metadata_per_page.to_json %>,
   "search": <%= metadata_search.to_json.html_safe %>,
+<% if params.has_key?(:include_permissions) %>
```

Comment:
Don't we need to make sure the model is authorizable?

Reply:
according to @ares 
> Not all resources includes Authorizable module. In fact they do only if they support granular filtering. In this case, I htink it would be safer to use authorized_for helper and pass it the url, which already contains this logic.

>
(in [here](https://github.com/theforeman/foreman/pull/5814#discussion_r211107786)) 

not all resources are `Authorizable` and thus `authorize_for` is used (in `can_create?`).
even though, if you think I am missing some edge cases, please let me know.

Thanks.

Reply:
Ok, so this is always true? Maybe we should just add tests for both cases (if they aren't already present, I didn't check) :-)

Reply:
Sure. It's actually a good idea :+1: 
@ares did we do something similar in our tests? I am trying to think of a way to test it. I mean I need to create or use a resource that has permissions but isn't `Authorizable` but have no clue how to do so - maybe you can save me some time and give me a hint?

Reply:
I don't think there's test like this. An example of a resource that does not support granular filtering but is protected by view_audit_logs permission on controller level is Audit. Another is HostClass (edit_classes permission). Another is Trend.